contrib/chatops/actions/workflows/post_result.yaml

```diff
@@ -7,41 +7,62 @@ input:
   - whisper
   - execution_id
   - channel
+  - context
 
 tasks:
-    format_execution_result:
-      action: chatops.format_execution_result
-      input:
-        execution_id: "{{ ctx('execution_id') }}"
-      next:
-        -
-          when: "{{ result().result.enabled == true }}"
-          publish:
-            - message: "{{ result().result.message }}"
-            - extra: "{% if 'extra' in result().result %}{{ result().result.extra }}{% else %}{}{% endif %}"
-          do:
-            - post_message
-        -
-          when: "{{ result().result.enabled == false }}"
-          do:
-            - result_disabled
+  # Use Python to get rid of PEP 414 unicode literals: u'string-value'
+  remove_unicode_literals:
+    action: core.local
+    input:
+      cmd: "python -c \"print \\\"{{ ctx('context') }}\\\"\""
```

Comment:
Hm, so context value here is a Python dictionary string?

EDIT: Looking below, it looks like it, yeah.

Reply:
I know ``context`` value will be provided by the chatops adapter which is, in theory, defined by developers, but I still think there is a potential of command injection here so we should try to "sanitize" / format this object in a safer manner (ideally on input).

Reply:
I removed this - it fixed the problem in the wrong way.